src/Illuminate/Support/Str.php

```diff
@@ -1034,7 +1034,7 @@ public static function flushCache()
      * @param  string|null  $encoding
      * @return string
      */
-    public static function mbStrPad($value, $length, $pad = ' ', $pad_type = STR_PAD_RIGHT, $encoding = null)
+    protected static function mbStrPad($value, $length, $pad = ' ', $pad_type = STR_PAD_RIGHT, $encoding = null)
```

Comment:
protected is still a new addition to the non-private API

Reply:
this whole PR is really a new feature anyway, so should target 9.x.

Reply:
Seems you're looking at an older version? He inlined that method after my comment.

Reply:
I'm marking this as resolved since the function was previously inlined. Don't hesitate to reopen it if need be.